script/build-icons

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+dir="$(cd "$(dirname "$0")" && cd "../src/common/icons" && pwd)"
```

Comment:
What did my past self do here. 🤦😁 

The following should do the same in a less indirect way:

```suggestion
dir="$(realpath "$(dirname "$0")../src/common/icons")"
```

Reply:
Was missing a `/` but works like charm ✨ (did not know about `realpath`)